### DISCUSSION TITLE
Colors are inconsistent in the gameplay mod pickers

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23960] [Ideas]
For an increased difficulty, the color of the multiplier should be red, but is also green in the song selection. The same happens for difficulty reduction but with the colors reversed.

![image](https://github.com/ppy/osu/assets/101510334/94e593dd-d59f-4f13-97ea-90b66a6a7081)
![image](https://github.com/ppy/osu/assets/101510334/c4628761-3ce0-4b39-8c5f-026f5f20ae62)

**Comment by bdach** [comment]
Song select is long overdue a redesign. Wouldn't bother fixing and just wait for the redesign to land.
